Signal/src/ViewControllers/OWSQRCodeScanningViewController.m

```diff
@@ -47,11 +48,33 @@ - (instancetype)initWithCoder:(NSCoder *)aDecoder
     return self;
 }
 
-- (void)viewDidLoad
+- (void)loadView
 {
-    [super viewDidLoad];
-    self.maskingView = [[UIView alloc] initWithFrame:self.view.frame];
-    [self.view addSubview:self.maskingView];
+    [super loadView];
+
+    OWSBezierPathView *maskingView = [OWSBezierPathView new];
```

Comment:
I think this is a nice example of why `OWSBezierPathView` is useful. Gathers concerns, lets you easily update path in response to layout.